extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +219,30 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (!connectionInfo?.password || includePassword === constants.noString) {
+								let userPassword = await vscode.window.showInputBox({
+									prompt: constants.includePassword,
+									placeHolder: constants.enterPasswordManually,
+									ignoreFocusOut: true,
+									password: true
+								});
+								if (!userPassword) {
+									// User will have to manually enter password later
+									let openSettingsFile = await vscode.window.showWarningMessage(constants.userPasswordLater, constants.openFile, constants.closeButton);
+									if (openSettingsFile === constants.openFile) {
+										// open local.settings.json file
+										await vscode.commands.executeCommand(constants.vscodeOpenCommand, vscode.Uri.file(localSettingsPath));
+									}
+								} else {
+									// replace the default password with the user's password
+									await vscode.commands.executeCommand(constants.vscodeOpenCommand, vscode.Uri.file(localSettingsPath));
```

Comment:
The await here is unnecessary. Although I'm curious why you're opening the file here anyways. (and regardless of why - you should probably only open it after replacing the password text since the user doesn't need to see it changed)